label.rb

```diff
@@ -0,0 +1,15 @@
+class Label
+  attr_reader :title, :color, items
```

Comment:
- [ ] Please make this line of code look like this:
`attr_reader :title, :color, :items`

Reply:
Thank you for your valuable comment, Already solved 🖌️